src/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -58,6 +58,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Arg_HTCapacityOverflow" xml:space="preserve">
+    <value>Capacity overflowed and went negative.</value>
```

Comment:
Something like "Cannot add more than {0} objects." might be more useful/user oriented? They don't care whether we use signed ints.

Reply:
I didn't address this one as we now should never hit this message unless the array entirely overflows.